android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -75,6 +75,10 @@ class CustomTabsToolbarFeature(
     internal var readableColor = Color.WHITE
     private var scope: CoroutineScope? = null
 
+    /** Gets the current custom tab session */
+    @VisibleForTesting
```

Comment:
```suggestion
    private val session: CustomTabSessionState? get() = sessionId?.let { store.state.findCustomTab(it) }
```
This doesn't need to be visible, but it's a nit, so let me fix this myself and then try to land this patch.